test/inductor/test_torchinductor_opinfo.py

```diff
@@ -413,8 +413,6 @@ def wrapper_noop_set_seed(op, *args, **kwargs):
     "addmv": {f16},
     "argsort": {b8, f16, f32, f64, i32, i64},
     "as_strided.partial_views": {f16},
-    "clamp_max": {b8},
-    "clamp_min": {b8},
```

Comment:
Aside from these two, you can remove `max.binary, min.binary, maximum, minimum` too

Reply:
Oh, just saw the comment. I will send a standalone PR to remove it.